### ISSUE TITLE
Sitemap

### ISSUE THREAD

**Issue by Shuunen** [Shuunen/bergerac-roads#17]
On doit établir le sitemap pour avoir une idée claire du site, des pages, imbrications, déroulés disponibles etc...

**Comment by LeGaby** [comment]
C'est quoi un sitemap exactement ?

**Comment by Shuunen** [comment]
Voici la liste des routes (je la mettrais à jour avec vos remarques) : 

- `/` la home
- `/domaine/chateau-chouette`

**Comment by Shuunen** [comment]
@LeGaby toutes les routes ou chemins empruntables par un utilisateur

**Comment by LeGaby** [comment]
Sur quelle page se fait la recherche ? La page d'accueil ?

**Comment by Shuunen** [comment]
Je pense oui à valider avec @bengrol

**Comment by Shuunen** [comment]
mais on peut avoir la recherche initiée sur la home et une page de résultat de recherche... ?

**Comment by LeGaby** [comment]
Oui, ou alors un bloc affichant les résultats qui apparaît sur la page d'accueil ?

**Comment by LeGaby** [comment]
Mais je pense que ton idée est plus intuitive, @Shuunen.

**Comment by Shuunen** [comment]
> Oui, ou alors un bloc affichant les résultats qui apparaît sur la page d'accueil ?
pourquoi pas oui :slightly_smiling_face: 

> je pense que ton idée est plus intuitive,
les deux me vont, ça va surtout dépendre de ce qu'on veut proposer à l'utilisateur en terme d'UX suite à sa recherche

**Comment by LeGaby** [comment]
@Shuunen , je suis d'accord avec ta liste des routes actuelles et à venir.

Je mettrais bien les infos de contact dans le footer, ça me semble un peu trop d'en faire une page.
Et de la même manière, le pourquoi du comment d'un site dans un bloc en haut de la page d'accueil.

Concernant l'organisation des pages entre domaines et vignobles, j'avoue ne pas avoir assez de données pour émettre une idée cohérente... On a les textes sur le Drive pour présenter les vignobles (Bergerac, Duras, etc) mais y a-t-il assez de choses à dire sur chaque domaine pour en faire une page ?

**Comment by Shuunen** [comment]
En effet si on a pas trop de contenu pour "a propos" et "contact" on les met direct dans la home et footer, à valider avec @bengrol 

Pour l'organisation des pages entre domaines et vignobles, pareil @bengrol sera notre guide spirituel :star2:

**Comment by bengrol** [comment]
Le mar. 29 mai 2018 à 13:16, Romain Racamier <notifications@github.com> a
écrit :

> En effet si on a pas trop de contenu pour "a propos" et "contact" on les
> met direct dans la home et footer, à valider avec @bengrol
> <https://github.com/bengrol>
>
>
OUI




Pour l'organisation des pages entre domaines et vignobles, pareil @bengrol
> <https://github.com/bengrol> sera notre guide spirituel 🌟
>
> —
>

un POI (chateaux etc ...) est un domaine
un domaine (region AOC) est un domaine viticole

**Comment by bengrol** [comment]
Si ton hypothèse c’est :
POI =domaine
AOC = vignoble 
C’est ok pour moi.

**Comment by Shuunen** [comment]
Ok dans ce cas quelle terminologie on utilise ?

**Comment by bengrol** [comment]
Pour les url ?

**Comment by Shuunen** [comment]
domaine et vignoble comme au dessus ? Ou domaine et région ? Ou autre ?

**Comment by Shuunen** [comment]
Histoire d'être le plus clair possible pour l'utilisateur et pour un bon référencement

**Comment by Shuunen** [comment]
@bengrol yes

**Comment by bengrol** [comment]
Domaine et vignoble me paraissent plus parlant d’un point de vu utilisateurs et référencement aussi. Donc, utilisons ceux là.

**Comment by Shuunen** [comment]
Nickel

**Comment by bengrol** [comment]
Normalement si. 

Domaine/chateau_de_mon_zguegue

Vignoble/bergerac

**Comment by Shuunen** [comment]
@LeGaby  un vignoble regroupe des domaines, on peut donc afficher une page `/vignoble/pecharmant` qui elle listera les domaines de cette région/vignoble comme `/domaine/chateau-chouette`

**Comment by Shuunen** [comment]
du coup je vous propose l'ajout de 2 nouvelles routes : 
- `/vignobles`  la liste des régions/vignobles de Bergerac
- `/domaines`  la liste des domaines de Bergerac de A à Z

**Comment by bengrol** [comment]
@Shuunen en faite la route /vignoble/pecharmant affichera une page de présentation avec le contenu en FR & UK qui est sur le drive.

**Comment by Shuunen** [comment]
je pensais à cette page comme un fallback par exemple si la personne va sur une url mal-formée genre `/vignoble/pecharmantZ` on pourrait afficher comme la page `/domaine/chateau-chouetteZ` : "Ce domaine n'est pas répertorié, vous pouvez voir notre [liste complète des domaines](#) ici ou revenir à la [page d'accueil](#)." 

et donc avoir la même chose avec les vignobles

je sais que ça arrive des clients qui copie/colle mal des liens et je trouve ça plus classe qu'une page 404 "Désolé, retour à l'accueil"

**Comment by bengrol** [comment]
Ah oui pas con.

**Comment by Shuunen** [comment]
Apparement il nous manquerait cote de duras

**Comment by bengrol** [comment]
On peut fermer cet issue ?

**Comment by Shuunen** [comment]
@bengrol les pages vignobles & domaines n'éxistent pas, à toi de voir s'il faut les créer (et du coup laisser cette issue ouverte) ou pas
